proxy/mtproto/auth.go

```diff
@@ -83,7 +83,7 @@ func generateRandomBytes(random []byte, connType [4]byte) {
 			continue
 		}
 
-		if 0x00000000 == (uint32(random[7])<<24)|(uint32(random[6])<<16)|(uint32(random[5])<<8)|uint32(random[4]) {
```

Comment:
The original coding style can be used to prevent accidental assignment. While this is redundant in this case, this practice should be considered to be a beneficial one. Additional justification required for this change.

Reply:
I can reject it, if you want.

Reply:
Rejected in 0401a91ef49befdd7d9dafbced973b211c97b616